<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro</class>
 <widget class="QDialog" name="registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/rebellion-wallpapers-2.jpg);</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>107</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/transparnt.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>ESCRIBA AQUI</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>166</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/transparnt.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>ESCRIBA AQUI</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>241</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/transparnt.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>ESCRIBA AQUI</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>300</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/transparnt.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>ESCRIBA AQUI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button_reg.png);
background-image: url(:/transparnt.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1865661-0 (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/warning.png);
background-image: url(:/transparnt.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/warning.png);
background-image: url(:/transparnt.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back1.png);
background-image: url(:/transparnt.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
